scripts/delete.sh

```diff
@@ -79,3 +79,8 @@ kustomize build default | kubectl delete -f - --ignore-not-found=true
 kustomize build runtimes --load-restrictor LoadRestrictionsNone | kubectl delete -f - --ignore-not-found=true
 kubectl delete -f dependencies/quickstart.yaml --ignore-not-found=true
 kubectl delete -f dependencies/fvt.yaml --ignore-not-found=true
+
+# Roll back to previous status
+if [[ -n $namespace ]]; then
+  kubectl config set-context --current --namespace="default"
```

Comment:
Is it possible to capture the previous namespace and reset to the specific name rather than "default"?

Reply:
@chinhuang007 , I think it is hard to trace in the uninstall stage(or we need caches some things to record this in the install stage), it is best to leave this to use to input. Currently, I think use the `default` is a good choice. Thanks

Reply:
I don't think we need to know the namespace used during installation. Can we just capture the current namespace when switching the context in L69-L71 and reset back to that namespace?

Reply:
Got it， please review this again， Thanks @chinhuang007